fbw-common/src/systems/instruments/src/EFB/Navigation/Navigation.tsx

```diff
@@ -537,6 +537,8 @@ export const ChartViewer = () => {
                   {aircraftIconVisible && boundingBox && (
                     <svg
                       ref={planeRef}
+                      width={boundingBox.planview.pixels.x2 - boundingBox.planview.pixels.x1}
```

Comment:
I'm not really sure why we need the width and height now, but without it the svg always had a size of 0x0 pixels.